### PR TITLE
feat: gsl sentinel landscape page

### PR DESCRIPTION
**Pull request by ZachBeck**
New website page for https://github.com/agrc/porter/issues/366

### CODE REVIEW

File: src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import GridForMoreResources from '@components/page/GridForMoreResources.astro';
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import { dataPages } from '@data/downloadMetadata';
+import { removeDateRange } from '@utils/data';
+
+export const metadata: IMetadata = {
+  pageTitle: 'Utah GSL Sentinel Landscape Boundary',
+  pageDescription: `Federally established boundary identifying the area of the Great Salt Lake Sentinel Landscape.`,
```
Comment (steveoh) [inline src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro:22]: This reads a bit better for me. Take it or leave it.

```suggestion
  pageDescription: `The federally established boundary identifying the area of the Great Salt Lake Sentinel Landscape.`,
```

File: src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  stewards: ['UGRC'],
+  type: ProductType.POLYGON,
+  category: SgidCategory.BOUNDARIES,
+  secondaryCategory: SgidCategory.PLANNING,
+};
+
+const page: IPageMetadata = {
+  ...metadata,
+  updateHistory: [],
```
Comment (steveoh) [inline src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro:31]: Update history requires a value.

```suggestion
  updateHistory: [`2024`],
```

File: src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      area where conservation efforts may align with the goals of the Sentinel Landscape Partnership to best leverage
+      funding opportunities.
+    </p>
+  </Fragment>
+
+  <Section title="Use the data" slot="downloads" titlePosition="center">
+    <HubDownloads {...page.hub} />
+  </Section>
+
+  <UpdateHistory slot="update-history" updates={page.updateHistory} />
```
Comment (steveoh) [inline src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro:57]: If this dataset is static then this component is not necessary
```suggestion
```

File: src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+  <Section title="More resources" slot="more-resources" titlePosition="center">
```
Comment (steveoh) [inline src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro:59]: If update history is not required then this section needs to have it's alternate style.
```suggestion
  <Section title="More resources" slot="more-resources" stripe titlePosition="center">
```

**Review by steveoh — CHANGES_REQUESTED** [review]
Looks good. A few items to take care of first.

File: src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import { dataPages } from '@data/downloadMetadata';
+import { removeDateRange } from '@utils/data';
+
+export const metadata: IMetadata = {
+  pageTitle: 'Utah GSL Sentinel Landscape Boundary',
+  pageDescription: `Federally established boundary identifying the area of the Great Salt Lake Sentinel Landscape.`,
+  stewards: ['UGRC'],
```
Comment (steveoh) [inline src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro:23]: Consider replacing our credit for veteran affairs or add both.
```suggestion
  stewards: ['Department of Veterans Affairs'],
```

File: src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  type: ProductType.POLYGON,
+  category: SgidCategory.BOUNDARIES,
+  secondaryCategory: SgidCategory.PLANNING,
```
Comment (steveoh) [inline src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro:26]: This secondary planning option does not match the SGID Index sheet. Please keep them synchronized.

Comment (ZachBeck) [reply src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro:26]: I updated the SGID index so it matches.

File: src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro (hunk 2 above)
Comment (steveoh) [reply src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro:30]: This still needs a value

File: src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+};
+
+const page: IPageMetadata = {
+  ...metadata,
+  updateHistory: [],
+  hub: {
+    ...dataPages[metadata.pageTitle],
+  },
```
Comment (steveoh) [inline src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro:33]: If you remove the slot="description" this object needs to know by adding a config: { skipInDepth: true },
```suggestion
  config: { skipInDepth: true },
  },
```

File: src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+      <CardWithPopularLink title="Sentinel Landscape Partnership" href="https://sentinellandscapes.org/"
+        >Working together to achieve landscape conservation and climate resilience goals.</CardWithPopularLink
+      >
+      <CardWithPopularLink title="Great Salt Lake" href="https://greatsaltlake.utah.gov/"
+        >Protecting and preserving the Great Salt Lake.</CardWithPopularLink
+      >
+    </GridForMoreResources>
+  </Section>
+
+  <Section title="You might also like" slot="you-might-also-like" stripe titlePosition="center">
```
Comment (steveoh) [inline src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro:67]: The stripes are there to alternate the section colors. There should not be consecutive stripe sections.
```suggestion
  <Section title="You might also like" slot="you-might-also-like" titlePosition="center">
```

**Review by steveoh — CHANGES_REQUESTED** [review]
Looks like the updateHistory was missed from the last round and a couple style fixes and this should be good to go.

**Review by steveoh — APPROVED** [review]

**Pull request merged**